Foundations/LisBella.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  LisBella.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 28/03/23.
+//
+
+import SwiftUI
+
+struct LisBella: View {
+    var body: some View {
+        ProfileCard(
+            title: "Conexões"
```

Comment:
Maybe this should be in English?

Reply:
App in Portuguese. At least in the prototype we made everything in Portuguese, if you want to change the app language, talk with our teachers